packages/odyssey-react-mui/src/labs/DataFilters.tsx

```diff
@@ -78,6 +86,10 @@ export type DataFilter = {
    * these are the options provided.
    */
   options?: Array<{ label: string; value: string }>;
+  /**
+   * A callback which renders a custom filter control
+   */
+  render?: (updateFilters: UpdateFiltersCallback) => ReactNode;
```

Comment:
Curious, what are we rendering in here?

Is this something we could do with Context or does this `render` function make more sense?

Reply:
This renders the filter control itself, which displays when the user clicks into the filter menu. By default, the control is simply a text input; if the `variant` is set, it'll use the variant instead. ("Variant" meaning one of the filter controls provided out-of-the-box, like a radio group or checkboxes).

I don't think a context would work well here; if I were the consumer, I'd expect this to be a callback in this format.